utilities/logger/src/LoggerInterface.ts

```diff
@@ -0,0 +1,9 @@
+interface LoggerInterface {
+  init(logger: any, options: any): void;
```

Comment:
If we research interfaces for other loggers we can notice that interfaces always have 2 levels of usage.

`Loogger.log(LoggerConst.Level_DEBUG,"Message")`

From this we can build actual helper methods for most common levels.
You can take look on: https://www.slf4j.org, which is exactly something we want to build here.